kitchen.yml

```diff
@@ -43,63 +43,66 @@ suites:
     attributes:
       osl-mysql:
         enable_percona_client: true
-  - name: multi-node-7
-    driver:
-      name: terraform
-      command_timeout: 1800
-      variables:
-          centos_image: 'CentOS 7.9'
-    provisioner: terraform
-    verifier:
-      name: terraform
-      systems:
-        - name: source
-          backend: ssh
-          hosts_output: source
-          sudo: true
-          user: centos
-          controls:
-            - all
-            - source
-          hostnames: source
-        - name: replica
-          backend: ssh
-          hosts_output: replica
-          sudo: true
-          user: centos
-          controls:
-            - all
-            - replica
-          hostnames: replica
-    excludes:
-      - centos_stream-8
-  - name: multi-node-8
-    driver:
-      name: terraform
-      command_timeout: 1800
-      variables:
-          centos_image: 'CentOS Stream 8'
-    provisioner: terraform
-    verifier:
-      name: terraform
-      systems:
-        - name: source
-          backend: ssh
-          hosts_output: source
-          sudo: true
-          user: centos
-          controls:
-            - all
-            - source
-          hostnames: source
-        - name: replica
-          backend: ssh
-          hosts_output: replica
-          sudo: true
-          user: centos
-          controls:
-            - all
-            - replica
-          hostnames: replica
-    excludes:
-      - centos-7
+          #  - name: multi-node-7
```

Comment:
Move the hash to the front of the line please. and also add a comment for why you're disabling it.

Reply:
I did not mean to leave those commented out for this commit. My bad.